examples/webserver/grpc/src/main/java/io/helidon/examples/webserver/grpc/GrpcMain.java

```diff
@@ -33,9 +36,24 @@ private GrpcMain() {
     public static void main(String[] args) {
         LogConfig.configureRuntime();
 
+        // initialize global config from default configuration
+        Config config = Config.create();
+        Config.global(config);
+        Config serverConfig = config.get("server");
+
+        // create a health check to verify gRPC endpoint
+        ObserveFeature observe = ObserveFeature.builder()
+                .addObserver(HealthObserver.builder()
+                                     .details(true)
```

Comment:
There are two ways to make custom health checks visible to Helidon. This is one of them (the one we've documented!). If the example adopts this technique (of creating the health observer explicitly, registering it, etc.) the health observer settings should be retrieved from config rather than details alone being hard-coded here. In fact the `application.yaml` file contains settings to control details but I see where this code would use it.

The second way is to implement the `HealthCheckProvider` SPI and its `List<HealthCheck> healthChecks(Config)` method. Helidon automatically discovers all such providers using service loading and invokes them. That allows Helidon's automatic discovery of the health observer to handle all the config and registration. Arguably quite a bit simpler.